tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -68,44 +74,49 @@ Feature: Sharing files and folders with internal groups
     Given user "user3" has logged in using the webUI
     When the user renames file "lorem.txt" to "new-lorem.txt" using the webUI
     And the user shares file "new-lorem.txt" with group "grp1" as "Editor" using the webUI
+    And user "user1" accepts the share "new-lorem.txt" offered by user "user3" using the sharing API
+    And user "user2" accepts the share "new-lorem.txt" offered by user "user3" using the sharing API
     And the user re-logs in as "user1" using the webUI
-    Then as "user1" the content of "new-lorem.txt" should not be the same as the local "new-lorem.txt"
+    Then as "user1" the content of "/Shares/new-lorem.txt" should not be the same as the local "new-lorem.txt"
     # overwrite the received shared file
-    When the user uploads overwriting file "new-lorem.txt" using the webUI
+    When the user opens folder "Shares" using the webUI
+    And the user uploads overwriting file "new-lorem.txt" using the webUI
     Then file "new-lorem.txt" should be listed on the webUI
-    And as "user1" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
+    And as "user1" the content of "/Shares/new-lorem.txt" should be the same as the local "new-lorem.txt"
     # unshare the received shared file
     When the user unshares file "new-lorem.txt" using the webUI
     Then file "new-lorem.txt" should not be listed on the webUI
     # check that another group member can still see the file
-    When the user re-logs in as "user2" using the webUI
-    Then as "user2" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
+    And as "user2" the content of "/Shares/new-lorem.txt" should be the same as the local "new-lorem.txt"
     # check that the original file owner can still see the file
-    When the user re-logs in as "user3" using the webUI
     Then as "user3" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
```

Comment:
```suggestion
    And as "user3" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
```

Good to see that actually these checks do not need a UI login of these other users.